server/src/main/java/com/ncc/neon/common/NlpClientQueryBuilder.java

```diff
@@ -0,0 +1,30 @@
+package com.ncc.neon.common;
+
+import org.springframework.http.HttpHeaders;
+
+public class NlpClientQueryBuilder {
+
+    public static HttpHeaders buildFileOperationQuery(String filename) {
+        HttpHeaders param = new HttpHeaders();
+        param.add("file", filename);
+        return param;
+    }
+
+    public static HttpHeaders buildFilePrefixQuery(String basename) {
+        HttpHeaders param = new HttpHeaders();
+        param.add("file_prefix", basename);
+        return param;
+    }
+
+    public static HttpHeaders buildTrainingOperationQuery(String trainSrc, String trainTgt, String validSrc, String validTgt) {
+        String basename = trainSrc.substring(0, trainSrc.length()-7);
```

Comment:
For now, would it be better to add a query parameter for the basename?  Then the builder wouldn't have to do the string parsing, and this will be refactored to the config blob method soon anyway.

Reply:
That's true, it'll be part of the config blob soon, so I guess whichever way is fine for now.

Reply:
I'll add a query parameter for now.